Lcd 016 n002 l

Misca1234 opened the pull request:
3D model for 

https://github.com/KiCad/Displays.pretty/pull/13

![bild](https://user-images.githubusercontent.com/25547797/27764092-26b2594a-5e91-11e7-9a8c-4a218221f235.png)

poeschlr commented:
Please move this model to Displays.3dshapes
and add a credits.md file (the later can be added in a separate pull request after all your lcd models are merged.)

poeschlr commented:
Take a look at your plated through holes. To get your copper pin color in the hole, increase the hole diameter of the green body by a small amount.

Misca1234 commented:
This push consist of two devices, the reason for this 
is that the old symbol "LCD12X2" had LCD-016N002L1" as an alias,
I could not just remvoe this alias becosue travis started to scan LCD12X2 (with errors)
so i had to "fix" LCD12X2 as well.
Which I discovered that LCD12X2 actuallay is actually WC1602A 

So I have 
Fixed LCD-016N002L1
Removed  LCD12X2
Added WC1602A

I have modified symbol, footprint, 3D models

LCD-016N002L1
http://www.vishay.com/docs/37299/37299.pdf

WC1602A
http://dl.btc.pl/kamami_wa/wc1602a0.pdf



![bild](https://user-images.githubusercontent.com/25547797/27769587-49098616-5f2d-11e7-89b7-948290c32377.png)

![bild](https://user-images.githubusercontent.com/25547797/27769592-4e8db68e-5f2d-11e7-8c06-9f8ee6ada88d.png)

Misca1234 commented:
And they are in Displays directory now

Misca1234 commented:
> Take a look at your plated through holes. To get your copper pin color in the hole, 
> increase the hole diameter of the green body by a small amount.

Uhm, do you want me to change so the "copper" is visible in the hole ?

poeschlr commented:
This was more meant as a tip not really a requirement. Your models look already awesome. 
(As soon as i merge the footprints i will probably merge the models as well. Currently i'm a bit too tired to make the final checks before i merge. Will do it tomorrow morning.)

Misca1234 commented:
Thanks a lot, I got the speed up now so,
Do you have a favourite I should add ?
Otherwise I will do an overhaul of the segments displays.

Also, some more colours should be nice
I would like to see 
one more darker white,
a darker green, so it can be used for PCB and the existing green for the display area
some better copper colour (currently I use gold pin colour as copper)
a medium grey
Perhaps a steel or aluminium colour (currently i use light grey pin colour)

PS
The pads are made up by 0.1 mm thick "disks ,so there could be some Z fighting, 
I will make them thicker next time and lower them into the pcb a bit to eliminate any potential Z fighting.
DS

poeschlr commented:
For LCD-016N002L the a and k pins are not aligned correctly.

Misca1234 commented:
Git messing around again, re pushed this push here
https://github.com/KiCad/packages3D/pull/81
